src/api/spec/services/notification_service/notifier_spec.rb

```diff
@@ -135,6 +135,25 @@
             end
           end
 
+          context 'and a group is subscribed to the event via RSS' do
+            before do
+              event_subscription = create(
+                :event_subscription,
+                eventtype: 'Event::RelationshipCreate',
+                receiver_role: 'any_role',
+                user: nil,
+                group: group,
+                channel: :rss
```

Comment:
As this scenario shouldn't be possible (I don't know yet what happened in production). I prefer to remove the spec. It was helpful to fix the issue but makes little sense to leave it.